controller:
  hero_chassis_controller:
    type: hero_chassis_controller/HeroChassisController
    front_left:
      p: 30.0
      i: 1.0
      d: 0.0
      target_velocity: 10.0

    front_right:
      p: 30.0
      i: 1.0
      d: 0.0
      target_velocity: 10.0

    back_left:
      p: 30.0
      i: 1.0
      d: 0.0
      target_velocity: 10.0

    back_right:
      p: 30.0
      i: 1.0
      d: 0.0
      target_velocity: 10.0

  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 1000


wheel_base: 0.4
track_width: 0.4
velocity: base
